src/pymor/operators/constructions.py

```diff
@@ -1362,3 +1362,55 @@ def __call__(self, U, mu=None):
         if self.raise_negative and np.any(norm_squared < 0):
             raise ValueError(f'norm is negative (square = {norm_squared})')
         return np.sqrt(norm_squared)
+
+
+class NumpyConversionOperator(Operator):
+    """Converts |VectorArrays| to |NumpyVectorArrays|.
+
+    Note that the input |VectorArrays| need to support
+    :meth:`~pymor.vectorarrays.interface.from_numpy`.
```

Comment:
Can it happen that `from_numpy` exists, but `to_numpy` does not?

Reply:
Yes. Neither method is guaranteed to work for an arbitrary `VectorArray`. The operator should only be used in 'safe' situations.

Reply:
so here one would have to add something like `or :meth:`~pymor.vectorarrays.interface.to_numpy, respectively` .